tools/travis/script.sh

```diff
@@ -30,7 +30,7 @@ set -e
 
 for suite in "${TESTS[@]}" ; do
   echo "Currently running test: $suite"
-  time nosetests --with-xunit --xunit-file=integration-test-results/$suite.xml --with-marvin --marvin-config=setup/dev/advanced.cfg test/integration/$suite.py -s -a tags=advanced,required_hardware=false --zone=Sandbox-simulator --hypervisor=simulator || true ;
+  time nosetests-2.7 --with-xunit --xunit-file=integration-test-results/$suite.xml --with-marvin --marvin-config=setup/dev/advanced.cfg test/integration/$suite.py -s -a tags=advanced,required_hardware=false --zone=Sandbox-simulator --hypervisor=simulator || true ;
```

Comment:
@weizhouapache should this be nosetests-3.4? We moved to python3, so we should ensure to install dependencies via pip3 and run tests using nosetests-3.4?

Reply:
@rohityadavcloud 
yes, the travis test still failed. it should be nosetest-3.4